docs/resources/default_namespace_settings.md

```diff
@@ -0,0 +1,26 @@
+---
+subcategory: "Settings"
+---
+
+# databricks_namespace_settings Resource
+
+The `databricks_default_namespace_settings` resource allows you to set the settings for the default namespace.
```

Comment:
Let's use the below as the explanation @hectorcast-db

          _The `databricks_default_namespace_settings` resource allows you to operate the setting configuration for the default namespace in the Databricks workspace.
          Setting the default catalog for the workspace determines the catalog that is used when queries do not reference
          a fully qualified 3 level name. For example, if the default catalog is set to 'retail_prod' then a query
          'SELECT * FROM myTable' would reference the object 'retail_prod.default.myTable'
          (the schema 'default' is always assumed).
          This setting requires a restart of clusters and SQL warehouses to take effect. Additionally, the default namespace only applies when using Unity Catalog-enabled compute._

Reply:
Updated the docs